ui/lib/pki/addon/routes/certificates/certificate/details.js

```diff
@@ -1,3 +1,22 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class PkiCertificateDetailsRoute extends Route {}
+export default class PkiCertificateDetailsRoute extends Route {
+  @service store;
```

Comment:
I think we've been extending the index routes for detail views so we get access to the store and secretMountPath but the routes here are pretty small/straightforward, potato potato 🥔

Reply:
Ya I'm not sure it's worth it just for service injections. There could be case where maybe down the road the `beforeModel` hook gets added to the index route and we don't want that functionality on the extended routes. Along the same lines I would extend if the parent route did something in the `beforeModel` hook that all of the extended routes relied on like pulling some data needed by the model hook for example.